tests/ship_test.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env python3
+
+from testUtils import Utils
+import testUtils
```

Comment:
Is this import necessary?  I don't see any use of anything from testUtils that isn't in `class Utils`, which is already imported.

Reply:
removed